### PR TITLE
chore: bump docsy to 0.7.0

### PR DESCRIPTION
**Pull request by MartinForReal**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: site/content/en/_index.html [hunk 1]
```diff
@@ -19,19 +19,19 @@
 </div>
 {{< /blocks/cover >}}
 
-{{< blocks/section >}}
+{{< blocks/section type="row" >}}
 
-{{% blocks/feature icon="fab fa-github" title="Contributions welcome!" url="https://github.com/kubernetes-sigs/cloud-provider-azure" %}}
+{{% blocks/feature icon="fa-brands fa-github" title="Contributions welcome!" url="/" %}}
```
Comment (lzhecheng) [inline site/content/en/_index.html:24]: Is it true that we don't need the repo url now?

Comment (MartinForReal) [reply site/content/en/_index.html]: updated. Thanks!

**Comment by MartinForReal** [comment]
<html>
<body>


/test pull-cloud-provider-azure-e2e-ccm-capz
--


<br class="Apple-interchange-newline">
</body>
</html>

**Pull request merged**